modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -909,7 +885,7 @@ impl Wallet {
             .enumerate()
         {
             let tx_hash = tx_hasher
-                .segwit_signature_hash(
+                .p2wsh_signature_hash(
```

Comment:
Note for reviewer: `SighashCache.segwit_signature_hash()` was [deprecated](https://docs.rs/bitcoin/0.31.2/bitcoin/sighash/struct.SighashCache.html#method.segwit_signature_hash) in bitcoin v0.31 and replaced with [`p2wpkh_signature_hash()`](https://docs.rs/bitcoin/0.31.2/bitcoin/sighash/struct.SighashCache.html#method.p2wpkh_signature_hash) and [`p2wsh_signature_hash()`](https://docs.rs/bitcoin/0.31.2/bitcoin/sighash/struct.SighashCache.html#method.p2wsh_signature_hash). The latter makes sense here.

Reply:
Sour code is identical too.